server.js

```diff
@@ -27,11 +22,105 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+
+if (process.env.RESET_DB) {
+  console.log('Database reset')
+  const seedDatabase = async () => {
+    await Author.deleteMany({})
+    await Book.deleteMany({})
+
+    // creating array with unique Authors from data
+    const uniqueAuthors = [... new Set(booksData.map(book => book.authors))]
+
+    await uniqueAuthors.forEach(async (authorName) => {
+
+      // Seeding authors from each unique author
+      const author = await new Author({ name: authorName }).save()
+
+      // Seeding books for each of the authors books 
+      await booksData.filter(book => book.authors === author.name).forEach(async (book) => {
+        await new Book({
+          title: book.title,
+          author: author, // unique author
+          average_rating: book.average_rating,
+          isbn: book.isbn,
+          isbn13: book.isbn13,
+          num_pages: book.num_pages,
+        }).save()
+      })
+    })
+  }
```

Comment:
Nice, find the books by this author and save them a this point.  A nice thought exercise would be to think "What if authors was an array of different authors"  How would I prevent duplicate books?